azure-pipelines.tools.yml

```diff
@@ -0,0 +1,12 @@
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: 'Install Node.js'
+
+- task: geeklearningio.gl-vsts-tasks-yarn.yarn-installer-task.YarnInstaller@3
```

Comment:
Do we need this dependency? Can we just directly install it with a shell command instead?

Reply:
Ken advised me on this approach. I'm open to other methods, as well.

Reply:
@jdhuntington - we could install it through shell, I've not tested how fast that is. This is just what we've been using for the other tasks, seems to "install" yarn pretty quickly and made available nicely without having to worry about installing something globally on a CI box. That's the only difference to me...